ilastik/applets/tracking/base/pluginExportOptionsDlg.py

```diff
@@ -194,8 +195,8 @@ def updateFromSlot(self):
             # Re-configure the slot in case we changed the extension
             self._filepathSlot.setValue(file_path)
 
-        if self._bdvFilepathSlot.ready():
-            bdv_file_path = self._bdvFilepathSlot.value
+        if self._additionalPluginArgumentsSlot.ready():
```

Comment:
Do you think this would work with a plugin that has a different key in `_additionalPluginArgumentSlot`? Is it somehow possible from the plugin to infer which keys to expect?

Reply:
:white_check_mark: